src/Renderer/ThreeExtended/Feature2Mesh.js

```diff
@@ -143,8 +128,8 @@ function prepareBufferGeometry(vert, color, altitude, extrude) {
 
 function featureToPoint(feature, properties, options) {
     // get altitude / color from properties
-    const altitude = getAltitude(options, properties, feature.vertices);
-    const color = getColor(options, properties);
+    const altitude = getProperty('altitude', options, 0, properties, feature.vertices);
+    const color = getProperty('color', options, randomColor, properties);
```

Comment:
You're passing a function (`randomColor`) instead of a value. Did you meant `randomColor()`?

Reply:
No, because we don't want to execute it if the options already have a value for it. Tiny optimization ;)

Reply:
Oh right, I missed the `if (typeof defaultValue === 'function') {` above. All good then :)